ios/RnExecutorch/models/StyleTransferModel.mm

```diff
@@ -49,10 +52,10 @@ - (UIImage *)postprocess:(UIImage *)input {
 
 - (UIImage *)runModel:(UIImage *)input {
   UIImage *processedImage = [self preprocess:input];
-  CGSize outputSize = {640, 640};
+  CGSize outputSize = {width_size, height_size};
   float* processedImageData = [ImageProcessor imageToFloatArray:processedImage size:&outputSize];
   
-  NSArray *modelInput = [self floatArrayToNSArray:processedImageData length:1228800];
+  NSArray *modelInput = [self floatArrayToNSArray:processedImageData length:(width_size * height_size * num_channels)];
   NSNumber* numInputs = [module getNumberOfInputs];
   NSLog(@"RnExecutorch: %@", [module getNumberOfInputs]);
   for (NSUInteger i = 0; i < [[module getNumberOfInputs] intValue]; i++) {
```

Comment:
Can you also get rid of that for loops that only log stuff there? They don't seem necessary, thank you for contributing!:D

Reply:
Sure, I'm wondering if this PR is still needed as there is a PR with style transfer using OpenCV